src/decorators/query.ts

```diff
@@ -28,32 +28,32 @@ export function query(options: TQueryOptions | TJSONSchema) {
     Reflect.defineMetadata(MetadataKey.ROUTE_QUERY, jsonSchema, target, key);
 
     before(async (req: Request, res: Response, next: NextFunction) => {
-      let query = req.query;
```

Comment:
Renamed due to tsline complaining about shadowing of the surrounding method. (same for `method` and `path` and one more I think)